python/tests/sql/test_dataframe_api.py

```diff
@@ -81,6 +81,7 @@
     (stf.ST_ExteriorRing, ("geom",), "triangle_geom", "", "LINESTRING (0 0, 1 0, 1 1, 0 0)"),
     (stf.ST_FlipCoordinates, ("point",), "point_geom", "", "POINT (1 0)"),
     (stf.ST_Force_2D, ("point",), "point_geom", "", "POINT (0 1)"),
+    (stf.ST_GeometricMedian, ("multipoint",), "multipoint_geom", "", "POINT(22.500002656424286 21.250001168173426)"),
```

Comment:
OK. There should be a space between `POINT` and `(`. It should be `POINT (`

Reply:
@jiayuasu Yeah, I figured. Should have installed python for testing. Thanks for quick reviews and your help.